### ISSUE TITLE
README needs updating

### ISSUE THREAD

**Issue by EleanorRobson** [oracc/nammu#217]
Raquel, when you get a minute could you update the README to:

* link to version 0.6 under "How to run Nammu"
* bring the list of current issues up to date (most of these are fixed now)

I'll then notify a core group of testers to download it and start playing :) Thanks!

**Comment by raquel-ucl** [comment]
Hi Eleanor,

You are right! I've changed that on the new branch which I wanted to polish before merging and have someone else from the team code review, but it hasn't been merged yet to the master branch. 

It's visible from this link:

https://github.com/oracc/nammu/tree/settings_gui

However I'd rather have the current branch reviewed and merged to master before sharing. Does that sound OK? I'll try to speed this up and update here when it's ready.

The list of issues is up to date, I think. The are some issues we descoped and labelled as such to review later, but the rest are still open because they haven't been completed. They have low priority, but I'm keeping there for when I have some time to work on them. Is there any one in particular you think should be closed?

**Comment by EleanorRobson** [comment]
Great, thanks! We can wait if you want to get the code reviewed first — I understand :)

Looking at the README again, at this list of features under development:
    * Line wrapping
    * JAR bundle and signature
    * Search/replace
    * Split view

It's fair to say that all but the second now work reasonably well. I don't want testers to think that they can't search & replace, or split the view, for instance. Maybe rephrase to say that these are features you'd particularly welcome feedback on?

e

**Comment by raquel-ucl** [comment]
Ah, got it. I thought you meant the list of issues on the issues tab, not the README. I'll update that in a minute. Thanks for spotting that!

**Comment by raquel-ucl** [comment]
Done! I have probably shown this to you before, but just in case, you can check the changes I've made on this diff:

https://github.com/oracc/nammu/pull/216/files#diff-04c6e90faac2675aa89e2176d2eec7d8

I think the README is getting too large and I should make a tidier user guide. I'll make an issue about that.

**Comment by raquel-ucl** [comment]
@EleanorRobson I've just merged #216 so the corrected README should be now visible and point to the latest version. 
Please let me know if there's something else I should change, or close the issue if you're happy with the changes.

**Comment by EleanorRobson** [comment]
Splendid, thank you! I've read it and it looks good. So let's sign off and start thinking about the weekend...
